src/operations/Completion.cpp

```diff
@@ -474,8 +360,11 @@ static bool deprecated(const Luau::AutocompleteEntry& entry, std::optional<lsp::
     return false;
 }
 
-static std::optional<lsp::CompletionItemKind> entryKind(const Luau::AutocompleteEntry& entry)
+std::optional<lsp::CompletionItemKind> WorkspaceFolder::entryKind(const Luau::AutocompleteEntry& entry)
```

Comment:
minor q: could we leave this as static and instead just pass a platform ptr to the function? I try not to keep increasing the surface area of WorkspaceFolder